deco-sites/otica-isabela
# updated all svgs to use currentColor, fix PLP Category Text on Admin …

…and fix typo

## What is this contribution about?

Update all SVGs to use "currentColor" prop.
Fix typo on ADMIN.
Fix CategoryText List on PLP ADMIN

## How to test it?

Check any of the Icons within the Store and check if they are appearing properly.
Check the Newsletter admin and look for any typo.
Check the PLP Category Text on Admin and see that is possible to add a label on each item.
